bin/twlight_docker_entrypoint.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+SECRET_KEY=twlight ./bin/twlight_static.sh
```

Comment:
I think putting this here means that we have to run the django collect static command any time we restart a container in production. Is that necessary?

Reply:
Yes, it would, which I don't like. 

Maybe we can pair up on this on Monday. 
I couldn't get the RTL static assets to generate unless I put it in the entrypoint script after the container has already started.